src/Maestro/Maestro.Web.Tests/TestDatabase.cs

```diff
@@ -65,7 +65,7 @@ public async Task<string> GetConnectionString()
         {
             string databaseName = $"{TestDatabasePrefix}_{TestContext.CurrentContext.Test.ClassName.Split('.').Last()}_{TestContext.CurrentContext.Test.MethodName}_{DateTime.Now:yyyyMMddHHmmss}";
             TestContext.WriteLine($"Creating database '{databaseName}'");
-            await using (var connection = new SqlConnection("Data Source=localhost\\SQLEXPRESS;Initial Catalog=master;Integrated Security=true"))
+            await using (var connection = new SqlConnection("Data Source=localhost\\SQLEXPRESS;Initial Catalog=master;Integrated Security=true")) // CodeQL [SM03452] This 'connection string' is only for the local SQLExpress instance and has no credentials, Encrypt=true is unnecessary
```

Comment:
QQ - What would happen if we added `Encrypt=true` for the local flows? Would there be any harm? Maybe it would get resolved like that as well?

Reply:
I just tried and I am having:
> System.ComponentModel.Win32Exception : The certificate chain was issued by an authority that is not trusted.

https://dev.azure.com/dnceng-public/public/_build/results?buildId=242945&view=logs&j=538cb63c-6e7c-5278-f82a-af130957b436&t=32d50a88-ec3c-51df-b813-951180b20f1d&l=639